ansible/overcloud-grafana-configure.yml

```
@@ -8,6 +8,30 @@
       group_by:
         key: "monitoring_with_grafana_enabled_{{ kolla_enable_grafana | bool }}"
 
+- name: Set the Monasca control plane project ID
+  hosts: monitoring_with_grafana_enabled_True[0]
+  gather_facts: False
+  tags:
+    - grafana
+  vars:
+    venv: "{{ virtualenv_path }}/shade"
+  roles:
+    - role: stackhpc.os-openstackclient
+      os_openstackclient_venv: "{{ venv }}"
+
+  tasks:
+    - name: Look up Monasca control plane project ID
+      shell: >
+        source {{ venv }}/bin/activate &&
+        openstack project show monasca --format json --column id
+      register: monasca_project_show
+      changed_when: False
+      environment: "{{ openstack_auth_env }}"
```

@ktibi asked if he could use this feature. As it stands, it depends on the monasca project existing. I think ideally we'd make these two tasks dependent upon a `kolla_enable_monasca` variable. It might be a bit misleading to add that as it stands as it wouldn't do anything. Alternatively we could add `failed_when: False` to ignore errors. What do you think?

---

Hmm. I like the idea of being explicit with ```kolla_enable_monasca```. If I put a big comment near it explaining that it's work in progress hopefully it shouldn't be too misleading?